recipes/_configure_authentication.rb

```diff
@@ -138,6 +138,7 @@
 end
 
 splunk_template 'system/authentication.conf' do
+  sensitive true
```

Comment:
How about `sensitive auth_stanzas.any? { |_, v| v.key? "bindDNpassword" }` since it's only sensitive if there's a bind password